client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -20,6 +21,7 @@ interface UrlQueryParams {
   filterKey?: string | string[];
   initialSort?: UrlQuerySort;
   filterCondition?: string;
+  additionalFilters?: { key: string; condition?: string; value?: string }[];
```

Comment:
So that we can have multiple conditions for different keys instead of just one condition for all

Reply:
now that there is a `filterKey` and `additionalFilters` this is getting messy. 
I'd prefer to remove `filterKey` and `filterCondition` and change over to use the new filter format that you've made. Here's my [suggestion branch](https://github.com/openmsupply/open-msupply/compare/2386-Implement-Date-and-Number-filter-in-frontend...2386-suggested) which does this

Reply:
Oooh a lot neater (: 

Thanks Mark~